pkg/store-client/client.go

```diff
@@ -103,6 +104,8 @@ func (c *c) ListArtifacts(ctx context.Context, project string) (ids []string, er
 
 	if res.StatusCode() == http.StatusNotFound {
 		errz.Fatal(usererror.Wrapm(ErrProjectNotFound, "upload to remote repository failed"))
+	} else if res.StatusCode() == http.StatusUnauthorized {
```

Comment:
Will this be shown in the logs? I guess not

e.g. on downloading...
```
ui/build       failed to download artifact (permission denied)
ui/build       ... doing a build instead ....
```

Reply:
![pull-unauthorized](https://user-images.githubusercontent.com/944919/196674346-27a08d1c-949e-4674-9260-e3000c24faad.png)

For current implementation this is the output of an unauthorized pull